src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -960,17 +960,17 @@ public function query_child_records( $args = array() ) {
 	 *
 	 * @return WP_Query|WP_Error|bool
 	 */
-	public function get_child_record_by_status( $status = 'success', $qty = -1, array $args = array() ) {
+	public function get_child_record_by_status( $status = 'success', $qty = -1, array $args = [] ) {
 		$statuses = Tribe__Events__Aggregator__Records::$status;
 
 		if ( ! isset( $statuses->{ $status } ) && 'trash' !== $status ) {
 			return false;
 		}
 
-		$args = array_merge( $args, array(
+		$args  = array_merge( $args, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 0787c7a9256127d906f7daa79b684943</sub>